index.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const knex = require("knex");
+const knexConfig = require("./knexfile.js");
+const server = express();
+server.use(express.json());
+const db = knex(knexConfig.development);
+
+const getDishes = tbl => (req, res) => {
+  db(tbl)
+    .then(data => {
```

Comment:
I'd prefer to be less specific with this function name since it's supposed to be for general use. I'd consider making a separate file for general helpers, and there I'd define this as:
```javascript
const getAll = tbl => (_, res) => {
  db(tbl)
    .then(sendOK(res)) // theoretical callback here
    .catch(servErr(res));
```

Reply:
and then the usage would be something along the lines of:
```javascript
const getDishes = getAll('dishes');
```
assuming you still want it stored in that variable.